src/openrct2/actions/StaffHireNewAction.hpp

```diff
@@ -55,15 +55,15 @@ DEFINE_GAME_ACTION(StaffHireNewAction, GAME_COMMAND_HIRE_NEW_STAFF_MEMBER, Staff
 private:
     bool _autoPosition = false;
     uint8_t _staffType = static_cast<uint8_t>(StaffType::Count);
-    uint8_t _entertainerType = ENTERTAINER_COSTUME::ENTERTAINER_COSTUME_COUNT;
+    uint8_t _entertainerType = static_cast<uint8_t>(EntertainerCostume::Count);
```

Comment:
I've just merged a change that means you can change the type of this class to the new enum

Reply:
Thanks. I was looking at that, but then saw the code << DS_TAG(_entertainerType) << down on line 99. Will the enum class be able to Serialise as before?

I'll try making that change.

Reply:
It should, that's exactly what Duncan's https://github.com/OpenRCT2/OpenRCT2/pull/12810 targeted

Reply:
Cool. I've changed this. There's still a place where I compare the type of a new/loaded entertainer to static_cast<uint8_t>(_entertainerCostume::Count), in order to check that it's valid. Is there a better way than `if (static_cast<uint8_t>(_entertainerType) >= static_cast<uint8_t>(EntertainerCostume::Count))` ?

Reply:
No you will still need to do that. It's not great but it's the only way to validate it fully

Reply:
Gotcha, thanks.